src/callkit.cs

```diff
@@ -77,6 +78,17 @@ public enum CXErrorCodeCallDirectoryManagerError : long {
 		UnexpectedIncrementalRemoval = 8,
 	}
 
+	[iOS (14,5), NoWatch, NoTV, NoMac]
+	[Introduced (PlatformName.MacCatalyst, 14, 5)]
+	[ErrorDomain ("CXErrorDomainNotificationServiceExtension")]
+	[Native]
+	public enum CXErrorCodeNotificationServiceExtensionError : long
+	{
+		Unknown = 0,
+		InvalidClientProcess = 1,
+		MissingNotificationFilteringEntitlement = 2
```

Comment:
Add a `,` to the end of every last enum member to reduce future diff noise

```suggestion
		MissingNotificationFilteringEntitlement = 2
```